package.json

```diff
@@ -122,57 +122,57 @@
     }
   },
   "dependencies": {
-    "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1",
-    "@fontsource/cabin": "^4.5.5",
-    "@fontsource/rubik": "^4.5.6",
-    "@fortawesome/fontawesome-svg-core": "^6.1.1",
-    "@fortawesome/free-brands-svg-icons": "^6.1.1",
-    "@fortawesome/free-regular-svg-icons": "^6.1.1",
-    "@fortawesome/free-solid-svg-icons": "^6.1.1",
-    "@fortawesome/react-fontawesome": "^0.1.18",
-    "@mui/icons-material": "^5.10.9",
-    "@mui/material": "^5.10.12",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@fontsource/cabin": "^4.5.10",
+    "@fontsource/rubik": "^4.5.14",
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/free-brands-svg-icons": "^6.3.0",
+    "@fortawesome/free-regular-svg-icons": "^6.3.0",
+    "@fortawesome/free-solid-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.12",
     "@node-steam/vdf": "^2.2.0",
     "@shockpkg/icon-encoder": "^2.1.3",
     "axios": "^0.26.1",
     "check-disk-space": "^3.3.1",
     "classnames": "^2.3.1",
-    "crc": "^4.1.1",
+    "crc": "^4.3.2",
     "discord-rich-presence-typescript": "^0.0.8",
     "electron-store": "^8.0.1",
     "electron-updater": "^5.0.1",
-    "filesize": "^8.0.7",
+    "filesize": "^10.0.6",
     "font-list": "^1.4.5",
-    "fs-extra": "^10.1.0",
+    "fs-extra": "^11.1.0",
     "fuse.js": "^6.6.2",
     "graceful-fs": "^4.2.10",
     "howlongtobeat": "^1.7.0",
-    "i18next": "^21.6.16",
-    "i18next-fs-backend": "^1.1.4",
-    "i18next-http-backend": "^1.4.0",
+    "i18next": "^22.4.11",
+    "i18next-fs-backend": "^2.1.1",
+    "i18next-http-backend": "^2.1.1",
     "ini": "^3.0.0",
     "plist": "^3.0.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-i18next": "^11.16.7",
-    "react-markdown": "^8.0.3",
-    "react-router-dom": "^6.3.0",
-    "recharts": "^2.1.14",
+    "react-i18next": "^12.2.0",
+    "react-markdown": "^8.0.5",
+    "react-router-dom": "^6.8.2",
+    "recharts": "^2.4.3",
     "sanitize-filename": "^1.6.3",
     "shlex": "^2.1.2",
-    "short-uuid": "^4.2.0",
-    "simple-keyboard": "^3.4.136",
+    "short-uuid": "^4.2.2",
+    "simple-keyboard": "^3.5.31",
     "source-map-support": "^0.5.21",
     "steam-shortcut-editor": "^3.1.1",
-    "systeminformation": "^5.15.0",
+    "systeminformation": "^5.17.12",
     "ts-prune": "^0.10.3",
-    "tslib": "^2.4.0"
+    "tslib": "^2.5.0"
   },
   "scripts": {
     "start": "vite",
     "codecheck": "tsc --noEmit",
-    "find-deadcode": "ts-prune --error",
+    "find-deadcode": "ts-prune | sed -e '/satisfies/,+1d'",
```

Comment:
Can't trigger the error in vscode. So i would asume it has something todo with webstorm and need to be configured there. So this should be reverted in my eyes.

Reply:
TS-Prune does not support the `satisfies` operator. This is not a configuration issue, it's simply because the thing is virtually dead (but no other alternatives exist, as far as I can tell)
You won't see any false-positifes right now since we don't use that operator anywhere right now. Try adding something like
```ts
export const myObject = {} satisfies Record<string, unknown>
```
anywhere, you'll see that TS-Prune will complain about 3 things in that line:
- The actual variable (`myObject`), which is valid
- `satisfies`
- The token directly after `satisfies` (in this case `Record`)

Reply:
Ok we don't use the `satisfies` keyword yet.
I discovered this [pacakge](https://www.npmjs.com/package/unimported) from the ts-prune repo where someone had the same problem. We might just switch to this and remove ts-prune ?

Reply:
I'm a little confused here, I've already tried out that package, discovered that it doesn't seem to work right, and documented those findings on Discord ([here](https://discord.com/channels/812703221789097985/812703424995000361/1083052712641429565)). You even replied to that message

I don't think this issue is going to be in ts-prune forever. If it's really as simple as updating one of their dependencies, *someone* there will get around to it at some point, and then we can revert this mess

Reply:
Oh ok forgot that. 
Testing here and it works for me. We just need to setup the config file correctly.
My `.unimportedrc.json` so far that gives reasonable results:

```json
{
  "rootDir": "./src",
  "preset": "node",
  "entry": [
    "src/**/*"
  ],
  "extensions": [
    ".js",
    ".jsx",
    ".ts",
    ".tsx",
    ".d.ts"
  ],
  "ignorePatterns": [
    "**/node_modules/**"
  ],
  "ignoreUnimported": [],
  "ignoreUnused": [],
  "ignoreUnresolved": []
}
```

Result:
```
  | 66 unresolved imports

   1 │ common/types
   2 │ common/types/gog
   3 │ common/types/legendary
   4 │ common/types/electron_store
   5 │ backend/utils/aborthandler/aborthandler
   6 │ backend/constants
   7 │ backend/main_window
   8 │ backend/wiki_game_info/wiki_game_info
   9 │ common/types/epic-graphql
  10 │ backend/logger/logger
  11 │ backend/tools
  12 │ backend/dialog/dialog
  13 │ backend/launcher
  14 │ backend/wiki_game_info/gamesdb/utils
  15 │ ../../tsconfig
  16 │ backend/__tests__/skip
  17 │ backend/updater
  18 │ backend/electron_store
  19 │ backend/config
  20 │ frontend/assets/epic-logo.svg
  21 │ frontend/assets/gog-logo.svg
  22 │ frontend/assets/heroic-icon.svg
  23 │ frontend/components/UI
  24 │ frontend/components/UI/StoreFilter
  25 │ frontend/state/ContextProvider
  26 │ frontend/helpers/electronStores
  27 │ frontend/helpers
  28 │ frontend/types
  29 │ frontend/components/UI/WebviewControls
  30 │ frontend/components/UI/Dialog
  31 │ frontend/assets/settings_icon_alt.svg
  32 │ frontend/assets/heroic_card.jpg
  33 │ frontend/components/UI/PopoverComponent
  34 │ frontend/screens/Library/components
  35 │ frontend/helpers/library
  36 │ frontend/hooks/hasProgress
  37 │ frontend/components/UI/ErrorComponent
  38 │ frontend/components/UI/Anticheat
  39 │ frontend/components/UI/StoreLogos
  40 │ frontend/hooks/hasStatus
  41 │ frontend/components/UI/WikiGameInfo/components/HowLongToBeat
  42 │ frontend/components/UI/WikiGameInfo/components/GameScore
  43 │ frontend/components/UI/UninstallModal
  44 │ frontend/assets/down-icon.svg
  45 │ frontend/assets/play-icon.svg
  46 │ frontend/assets/stop-icon.svg
  47 │ frontend/assets/stop-icon-alt.svg
  48 │ frontend/components/UI/ActionIcons
  49 │ frontend/screens/WineManager/components/WineItem
  50 │ frontend/components/UI/ExternalLinkDialog
  51 │ frontend/components/UI/LanguageSelector
  52 │ frontend/components/UI/ThemeSelector
  53 │ frontend/hooks/useSetting
  54 │ frontend/helpers/gamepad
  55 │ frontend/components/UI/TwoColTableInput
  56 │ frontend/screens/Settings/SettingsContext
  57 │ frontend/components/UI/ProgressDialog
  58 │ frontend/components/UI/Collapsible/Collapsible
  59 │ frontend/components/UI/TextWithProgress
  60 │ frontend/hooks/useSettingsContext
  61 │ frontend/components/UI/ToggleSwitch
  62 │ frontend/assets/info_icon.svg
  63 │ frontend/assets/denied_icon.svg
  64 │ frontend/assets/rounded_checkmark_icon.svg
  65 │ frontend/components/UI/FormControl
  66 │ frontend/components/UI/UpdateComponent

 │ 5 unused dependencies

   1 │ @fontsource/cabin
   2 │ @fontsource/rubik
   3 │ source-map-support
   4 │ ts-prune
   5 │ tslib
```

Reply:
Ok i think i am close to solve this. Can i push it ?

Reply:
Unimported does not do quite the same thing as ts-prune, as it only checks for unimported *files*. I could still add new unused types to `common/types.ts` (for example) without it erroring out

Reply:
Ok yes i see. Threfore unimported tells you which third party modules are not used.The problem i see is that ts-prune is kind of deprecated and a new maintained tool would be nice

Reply:
Sure, but we're actively sacrificing functionality when switching to Unimported

Don't get me wrong, I'd absolutely prefer an alternative, but isn't the main point of having these tools that you get per-export analysis, instead of just per-file? It's easy to leave some variable in a file exported, but a whole *file* being unused sounds unlikely